docs/v8-new-performance-apis.md

```diff
@@ -0,0 +1,250 @@
+# New Performance APIs in v8
+
+> [!WARNING]
+> This document is WIP. We are working on this while we are preparing v8.
+
+In v8.0.0, we moved to new performance APIs. These APIs have been introduced in v7, so they can already be used there.
+However, in v8 we have removed the old performance APIs, so you have to update your manual instrumentation usage to the
+new APIs before updating to v8 of the JavaScript SDKs.
+
+## Why?
+
+In v8 of the JavaScript SDKs, we made the move to base the performance instrumentation of all Node-based SDKs to use
+[OpenTelemetry](https://opentelemetry.io/) under the hood. This has been done to better align with the broader
+ecosystem, and to allow to use common auto instrumentation packages to be able to cover more ground in the ever-changing
+JavaScript landscape.
+
+Since the way that OpenTelemetry works differs from how the SDK used to work, this required some changes in order to be
+compatible.
+
+Note that for Browser- or Edge-based SDKs, we are not (yet) using OpenTelemetry for auto instrumentation. However, in
+order to keep the SDKs isomorphic - especially for SDKs for Meta-Frameworks like Next.js, Sveltekit or Remix - we made
+the decision to align the performance APIs for all JavaScript-based SDKs.
+
+## The "old" Way of Manual Performance Instrumentation
+
+Previously, there where two key APIs for adding manual performance instrumentation to your applications:
+
+- `startTransaction()`
+- `span.startChild()`
+
+This showed the underlying data model that Sentry was originally based on, which is that there is a root **Transaction**
+which can have a nested tree of **Spans**.
+
+## The new model: Goodbye Transactions, Hello Spans Everywhere!
+
+In the new model, transactions are conceptually gone. Instead, you will _always_ operate on spans, no matter where in
+the tree you are. Note that in the background, spans _may_ still be bundled up to transaction for the UI. However, this
+happens transparently, and from an SDK perspective, all you have to think about are spans.
+
+## The Span schema
+
+Previously, spans & transactions had a bunch of properties and methods to be used. Most of these have been removed in
+favor of a slimmer, more straightforward API, which is also aligned with OpenTelemetry Spans. You can refer to the table
+below to see which things used to exist, and how they can/should be mapped going forward:
+
+| Old name              | Replace with                                         |
+| --------------------- | ---------------------------------------------------- |
+| `traceId`             | `spanContext().traceId`                              |
+| `spanId`              | `spanContext().spanId`                               |
+| `parentSpanId`        | Same                                                 |
```

Comment:
l: "Same" as in unchanged? Then I'd prefer that as I previously confused Same with applying the replacement value from the line above (maybe a me problem 😅)

Reply:
```suggestion
| `parentSpanId`        | Unchanged                                                 |
```